packages/core/src/contracts/newsroom.ts

```diff
@@ -124,6 +124,19 @@ export class Newsroom extends BaseWrapper<NewsroomContract> {
     return this.multisigProxy.isOwner(who);
   }
 
+  public editors(): Observable<EthAddress> {
+    return this.instance
+      .RoleAddedStream({ role: Roles.Editor }, { fromBlock: 0 })
+      .map(e => e.args.grantee)
+      .concatFilter(async e => this.isEditor(e));
+  }
+
+  public reporters(): Observable<EthAddress> {
```

Comment:
I imagine we're removing this soon

Reply:
I think we might still have reporters on the smart contract, it just makes sense to me. That way a reporter can both accept a position, and quit their position, if they want.

Reply:
we should meet about this soon